src/components/Cart/Cart.tsx

```diff
@@ -0,0 +1,52 @@
+import "./Cart.scss";
+import empty from "../../assets/illustration-empty-cart.svg";
+import Delivery from "../../assets/icon-carbon-neutral.svg";
+import { useContext, useEffect, useState } from "react";
+import { CartContext } from "../../context";
+import Item from "../CartItem/CartItem";
+
+function Cart({showModal}:{showModal: (arg: boolean)=> void}) {
+  const cartItems = useContext(CartContext)
+  const [Total, setTotal] = useState(0);
+
+  useEffect(() => {
+    if(cartItems){
+      let cartTotal = 0
+      cartItems.cart.forEach(item => {
+        cartTotal += item.price * item.quantity
+        setTotal(cartTotal)
+      });
+    }
```

Comment:
```suggestion
    if(!cartItems){
    return}
    else{
      let cartTotal = 0
      cartItems.cart.forEach(item => {
        cartTotal += item.price * item.quantity
        setTotal(cartTotal)
      });
    }
```
pros: easier to read and early return 💯